src/script/conversation/ConversationRepository.ts

```diff
@@ -1943,7 +1943,13 @@ export class ConversationRepository {
         if (conversationEntity) {
           // Check if conversation was archived
           previouslyArchived = conversationEntity.is_archived();
-
+          if (
+            previouslyArchived &&
+            conversationEntity.status() === ConversationStatus.PAST_MEMBER &&
+            type === CONVERSATION_EVENT.MEMBER_JOIN
+          ) {
+            this.unarchiveConversation(conversationEntity, false, 'event from notification stream');
```

Comment:
'event from notification stream'
Can we make it a constant?

Reply:
i added it to the class as a static member